try.html

```diff
@@ -830,10 +830,14 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
     <td><img src='/gitter/room/nwjs/nw.js.svg' alt=''/></td>
     <td><code>https://img.shields.io/gitter/room/nwjs/nw.js.svg</code></td>
   </tr>
-  <tr><th> JIRA issue: </th>
-    <td><img src='/jira/issue/https/issues.apache.org/jira/KAFKA-2896.svg' alt=''/></td>
+  <tr><th data-keywords='jira-issues'> JIRA issue: </th>
```

Comment:
Badge search already reads the header. I believe adding `data-keywords` here is not needed.